spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/VaultEnvironmentProperties.java

```diff
@@ -102,12 +103,12 @@ public void setScheme(String scheme) {
 		this.scheme = scheme;
 	}
 
-	public String getBackend() {
```

Comment:
There is a process to deprecate properties that spring boot has. We'll need to use it here. I'm on mobile, but if you haven't found it by Monday, ping me and I'll find it.

Reply:
I'll look into it, thanks.